webapp/crashstats/crashstats/tests/testbase.py

```diff
@@ -2,54 +2,11 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-import inspect
-from unittest import mock
-
 from django.contrib.auth.models import User
 import django.test
 
-import crashstats.crashstats.models
-import crashstats.supersearch.models
-
-
-classes_with_implementation = []
-for module in (crashstats.crashstats.models, crashstats.supersearch.models):
-    for _, klass in inspect.getmembers(module, inspect.isclass):
-        if issubclass(klass, crashstats.crashstats.models.SocorroMiddleware):
-            # Remember, the default thing for SocorroMiddleware is
-            # that it always has a class attribute called `implementation`
-            # it's by default set to None.
-            if klass.implementation:
-                classes_with_implementation.append(klass)
-
 
 class DjangoTestCase(django.test.TestCase):
-    def setUp(self):
```

Comment:
I removed all the implementation class mocking. It was a tremendous undertaking, but it was the only way to verify that the configman-less code worked right.